streams/src/test/java/org/apache/kafka/streams/state/KeyValueStoreTestDriver.java

```diff
@@ -191,6 +193,10 @@ public Sensor addLatencySensor(String scopeName, String entityName, String opera
         @Override
         public void recordLatency(Sensor sensor, long startNs, long endNs) {
         }
+        @Override
+        public Map<MetricName, ? extends Metric> metrics() {
+            return null;
```

Comment:
I know this is only a test class, but returning `Collections.emptyMap()` would be the nicer thing to do here